.github/workflows/issue_stale.yml

```diff
@@ -0,0 +1,29 @@
+name: Stale Issue Labeler
+
+on:
+  schedule:
+    - cron: "30 1 * * *"
+  workflow_dispatch:
+
+permissions:
+  issues: write
+  pull-requests: write
+  actions: write
+
+jobs:
+  issues:
+    name: Check for stale issues
+    runs-on: ubuntu-latest
+    if: ${{ contains(github.repository, 'BassT23/Proxmox') }}
+    steps:
+      - uses: actions/stale@28ca1036281a5e5922ead5184a1bbf96e5fc984e # v9.0.0
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Don't forget to match that !

Reply:
Nice to see :)

please pull it to develop, not the stable ;)